core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -250,7 +221,7 @@ protected:
     }
     objc::msg_send<void>(m_window, "center"_sel);
 
-    return {};
+    return m_window_show();
   }
```

Comment:
Standardised `m_window_show` semantic across all platforms and moved logic into there.